packages/rich-text/src/component/index.native.js

```diff
@@ -178,7 +178,7 @@ export class RichText extends Component {
 	}
 
 	onFormatChange( record ) {
-		const { start, end, activeFormats = [] } = record;
+		const { start = 0, end = 0, activeFormats = [] } = record;
```

Comment:
It looks like this is the only substantive change in this PR and the rest is chunk below this that shows up in the diff is just changes to the how the code is formatted, is that right?

Reply:
Yes, correct. I added the note to description few moments ago 

>The rest of the changes are autogenerated by prettier/linter.